### DISCUSSION TITLE
关于奇小怪的设定以及为它开发一套表情包的提议

### DISCUSSION THREAD

**Discussion by qihuju** [qihuju/qihuju.github.io#3] [Ideas]
![qihuju_small](https://user-images.githubusercontent.com/80709287/111905666-ae343b80-8a44-11eb-89b3-3b267b8eaad6.jpg)

提议：为它做一套奇忽局专属表情包（静态/动态，吃惊/开心/鄙视/666/喜欢等等），并且可以在微信/Line/Telegram等社交媒体应用里传播推广

设定如下（待议，待修改）：

> 全名：奇点小黑洞
> 昵称：奇小怪
> 种族：黑洞
> 特性：物质吸收（什么都能吃）
> 职务：奇忽局特派员（萌宠）
> 口头禅：这个可以吃不？

**Comment by Lupingyue** [comment]
非常同意这个提议。看看其它人有什么建议？

**Comment by qihuju** [comment]
update:

目前已经有了它的第一个二创作品，由 @黄嘉欣 Chlor Huang 创作：

![Screenshot 2021-03-21 at 23 38 09](https://user-images.githubusercontent.com/80709287/112166786-dad48880-8be7-11eb-827f-31ceb99bd371.png)

@Tz 给它写了一小段同人配文，如下

> 奉献
> 
> ……我不知道祂从而何来，也不知道为何在我面前现身，当祂紧闭的瞳孔前纠缠在一起的触手开始蠕动，我的脑海中有一声非人的低语响起：“奉献……”我的内心突然充满了欣快感，仿佛沐浴在神启之下，又仿佛被天使的翅羽轻轻抚摸着脸颊，眼前的一切也被多彩的光晕包围。祂的瞳孔仍然没有张开，身体周边却沸腾起来，鼓涌出一道道的藤蔓，那藤蔓如火焰般向四周探索、伸展，渐渐向我靠近。“奉献……”我聆听着，哭泣着张开双臂，去拥抱这份恩赐，去拥抱这份来自上天的恩典。七彩的藤蔓将我包围，我被巨大的喜悦和快感灌注全身，这一刻，我的灵与肉便是祂的索取，我即是祂，祂即是我……当我再度清醒时，祂已经消失了，但我知道，祂与我同在。“奉献……”我喃喃自语，举起双手，我的手指沸腾起来，那鼓涌出的一道道藤蔓，仿佛火焰般向四周探索、伸展。奉献……

**Reply by Lupingyue** [reply]
这段短文应该放到官网上啊。

**Reply by qihuju** [reply]
> 这段短文应该放到官网上啊。

已加入奇忽局艺术作品集：[点击查看](https://qihuju.wiki/docs/qihuju-artefact.html)
